sbi/inference/sre/sre.py

```diff
@@ -38,29 +38,30 @@ def __init__(
     ):
         """Sequential Ratio Estimation
 
-        As presented in _Likelihood-free MCMC with Amortized Approximate Likelihood Ratios_ by Hermans et al., Pre-print 2019, https://arxiv.org/abs/1903.04057
+        As presented in _Likelihood-free MCMC with Amortized Approximate Likelihood
+         Ratios_ by Hermans et al., Pre-print 2019, https://arxiv.org/abs/1903.04057
 
         See NeuralInference docstring for all other arguments.
 
         Args:
             classifier: Binary classifier
             num_atoms: Number of atoms to use for classification.
-                If -1, use all other parameters in minibatch
+                If -1, use all other thetas in minibatch
```

Comment:
This is really offtopic, but I think we should use `None`, not `-1` for these special cases.